implementations/rust/ockam/ockam_command/src/project/create.rs

```diff
@@ -16,11 +16,11 @@ use crate::{space, CommandGlobalOpts};
 /// Create projects
 #[derive(Clone, Debug, Args)]
 pub struct CreateCommand {
-    /// Name of the space the project belongs to.
+    /// Name of the Space the project belongs to.
     #[arg(display_order = 1001)]
     pub space_name: String,
 
-    /// Name of the project.
+    /// Name of the project - must be unique within parent Space
```

Comment:
```suggestion
    /// Name of the project - must be unique within parent Space.
```

Reply:
End-of-line points are trimmed, so there is no need to add it.